File: 프로그래머스/0/181852. 뒤에서 5등 위로/뒤에서 5등 위로.java
import java.util.*;
class Solution {
    public int[] solution(int[] num_list) {
        
        Arrays.sort(num_list);
        
        int [] arr = new int[num_list.length-5];
        int count = 0;
        for(int i = 5; i < num_list.length; i++){
            arr[count] = num_list[i];
            count++;
        }
        int[] answer = arr;
        return answer;
        
    }
}

File: 프로그래머스/0/181852. 뒤에서 5등 위로/뒤에서 5등 위로.cs
using System;

public class Solution {
    public int[] solution(int[] num_list) {
        Array.Sort(num_list);
        int[] answer = new int[num_list.Length - 5];
        int index = 0;
        for(int i = 5; i < num_list.Length; i++){
            answer[index++] = num_list[i];
        }
        return answer;
    }
}